cpu/esp32/Makefile

```
@@ -13,6 +13,10 @@ ifneq (, $(filter esp_bootloader, $(USEMODULE)))
     DIRS += bootloader
 endif
 
+ifneq (, $(filter esp32_bootloader_reset, $(USEMODULE)))
+    DIRS += esp-bootloader-reset
+endif
```

Why is it separate modules. Couldn't it be just a file in `cpu/esp32` which is selected in `cpu/esp32/Makefile`:
```suggestion
ifneq (, $(filter usb_board_reset, $(USEMODULE)))
  SRC += reset.c
endif
```
As separate module it would also require the extension of Kconfig.